packages/utxo-lib/tests/__fixtures__/transaction/litecoin.ts

```diff
@@ -31,6 +31,133 @@ export default {
             virtualSize: 125,
             weight: 500,
         },
+        {
+            description: 'Tx without advanced transaction marker and with mweb flag',
```

Comment:
If I'm correct in the previous comment, the mweb flag isn't really a flag without the marker, but there is ofc no need to change the fixture description.

Reply:
Good idea, changed fixture to "Tx without advanced transaction marker and with byte that could be misinterpreted as mweb flag"